CHANGELOG.md

```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
 - Clean up code by using Swift 5.7 capabilities. 
 - `.pagerTabItem(tag: SelectionType)` modifier now requires tag parameter which identifies the pager tab item. 
 - `pagerTabStripViewStyle(_ style: PagerStyle)` is the new modifier to set up pager style. 
-- Selection value, which indicates wich is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
-- Fix scroll behaivour when pager is added inside another pager or scrllable container. Whenever child page reatch its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
+- Selection value, which indicates which is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
+- Fix scroll behavior when pager is added inside another pager or scrollable container. Whenever child page reaches its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
 - Added several complex examples showing new functionality. 
 - Scrollable style page now uses iOS 16 native layout engine through Layout protocol.
-- Fixed inicator initial animation and position issue when presenting the page. 
-- Fixed pages reording, adding, deleting. Now it works perfectly. 
-- `public func onPageAppear(perform action: (() -> Void)?) -> some View` was removed. You should use onAppear native calback or useg selection state variable.
+- Fixed indicator initial animation and position issue when presenting the page. 
+- Fixed errors on reordering, adding and deleting pages. Now it works perfectly. 
```

Comment:
Resolved errors when reordering, adding, and deleting pages. Now it works perfectly.